src/s11/handlers/s11_DS_resp_handler.c

```diff
@@ -42,7 +43,10 @@ s11_DS_resp_handler(MsgBuffer* message, GtpV2MessageHeader* hdr)
 	/*Check whether has teid flag is set.
 	 * Also check whether this check is needed for DSR.
 	 * */
-	dsr_info.ue_idx = hdr->teid;	
+	//dsr_info.ue_idx = hdr->teid;	
```

Comment:
to keep things safe..Can we use ue_idx only when hdr->Teid is 0 ? Though current changes look good...add log if hdr->Teid != 0  && (hdr->Teid != ue_index). 

This will happen rarely when there is cross connection of tunnels.  You may do it or skip this.